### PR TITLE
Modified to use async wait / await to allow functionality with latest…

### PR DESCRIPTION
**Pull request by loredous**
… aiohttp requirements

### CODE REVIEW

**Comment by loredous** [comment]
Tested and functioning with HomeAssistant 0.65.6 / aiohttp 3.1.0

File: pyxeoma/xeoma.py [hunk 1]
```diff
@@ -74,30 +74,30 @@ def async_fetch_image_data(self, image_name, username, password):
...
     @asyncio.coroutine
-    def async_get_image_names(self):
+    async def async_get_image_names(self):
         """
             Parse web server camera view for camera image names
         """
 
         cookies = self.get_session_cookie()
         try:
             with aiohttp.ClientSession(cookies=cookies) as session:
```
Comment (jeradM) [inline pyxeoma/xeoma.py:96]: Missing `async`

**Comment by jeradM** [comment]
Thanks for this! See one comment

**Review by jeradM — CHANGES_REQUESTED** [review]

File: pyxeoma/xeoma.py (hunk 1 above)
Comment (jeradM) [reply pyxeoma/xeoma.py:96]: Actually don't worry about it. I'll take care of this when I create the release. Thanks again

**Review by jeradM — APPROVED** [review]

**Pull request merged**